assets/src/dashboard/app/api/useStoryApi.js

```diff
@@ -154,12 +168,71 @@ const useStoryApi = (dataAdapter, { editStoryURL, wpApi }) => {
     [wpApi, dataAdapter, editStoryURL]
   );
 
+  const trashStory = useCallback(
+    async (story) => {
+      try {
+        const response = await dataAdapter.deleteRequest(
+          `${wpApi}/${story.id}`,
+          {
+            data: story,
+          }
+        );
+        dispatch({
+          type: STORY_ACTION_TYPES.TRASH_STORY,
+          payload: reshapeStoryObject(editStoryURL)(response),
```

Comment:
Nit, do you need to passed in a completely reshaped object?  Or just the story Id?